### PR TITLE
Spacing and typography

### PR DESCRIPTION
**Pull request by niuage**

### CODE REVIEW

**Comment by niuage** [comment]
[Rails Sass update / Compass Gem Removal](https://trello.com/c/BDOMgPRy/5967-rails-sass-update-compass-gem-removal)

File: app/assets/stylesheets/reimagine2/devpost/_spacing.sass [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// Documentation: https://bulma.io/documentation/helpers/spacing-helpers/
```
Comment (devpost-mzheng) [inline app/assets/stylesheets/reimagine2/devpost/_spacing.sass:1]: is this new stuff or a replacement?

File: app/assets/stylesheets/reimagine2/devpost/_typography.sass [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@for $i from 1 through length($greys)
```
Comment (devpost-mzheng) [inline app/assets/stylesheets/reimagine2/devpost/_typography.sass:1]: is this new stuff or a replacement?

File: app/assets/stylesheets/reimagine2/devpost/_spacing.sass (hunk 1 above)
Comment (niuage) [reply app/assets/stylesheets/reimagine2/devpost/_spacing.sass:1]: That replaces the _spacing.sass that's in platform, where I manually added a few helpers that I needed.

File: app/assets/stylesheets/reimagine2/devpost/_typography.sass (hunk 2 above)
Comment (niuage) [reply app/assets/stylesheets/reimagine2/devpost/_typography.sass:1]: That's new. I want to have a few simple helpers for spacing and typography, matching our style guide, so we dont have to have custom css whenever we want some faded text.

**Pull request merged**